### ISSUE TITLE
fine-tune Pegasus with xsum using Colab but generation results have no difference

### ISSUE THREAD

**Issue by harrywang** [huggingface/transformers#10525]
Hi. I tried to fine-tune pegasus large with xsum dataset using Colab (Pro). I was able to finish the fine-tuning with batch size 1, and 2000 epochs in about 40 minutes (larger batch size crashed colab). The working Colab notebook I used is shared at https://colab.research.google.com/drive/1RyUsYDAo6bA1RZICMb-FxYLszBcDY81X?usp=sharing

However, the generated summary seems to be the same for the pegasus large model (https://huggingface.co/google/pegasus-large) and the fine-tuned model. But the generated result using pegasus xsum model (https://huggingface.co/google/pegasus-xsum) is different and much better. 

The training loss is already 0 and I am not sure what I have done wrong. Any help and pointers are highly appreciated. 

@sshleifer

**Comment by NielsRogge** [comment]
Hi,

could you please ask this question on the [forum](https://discuss.huggingface.co/)? We're happy to help you there!

Questions regarding training of models are a perfect use case for the forum :) for example, [here](https://discuss.huggingface.co/search?q=pegasus) you can find all questions related to fine-tuning PEGASUS.

**Comment by harrywang** [comment]
@NielsRogge thanks a lot! I will post the question in forum 😄🤝

**Comment by harrywang** [comment]
I think this issue can be closed - I used another input text and the generated text is different - I guess the fine-tuned model is different but for some input text, the generated result is exactly the same as the large model - interesting to know why though :).
